.gitignore

```diff
@@ -4,6 +4,8 @@ dbquit.m
 .lock
 # matlabpref backup
 .matlabprefs.mat.backup
+# Temp folders
+temp/
```

Comment:
In this pull request, used by PathManager for it's temp file. Figured general enough to include via folder rather than excluding individual files in future. This ignores *any* temp folder in repo (not just at root).

Reply:
its* temp file